front-end/src/components/BillingView.js

```diff
@@ -1,16 +1,18 @@
 import React, { Component } from 'react';
-import { connect } from 'react-redux';
+import CheckoutForm from './CheckoutForm';
+import { Elements, StripeProvider } from 'react-stripe-elements';
 
-class BillingView extends Component {
+export default class BillingView extends Component {
```

Comment:
Nice! If you are going to do this, you might as well just make it a functional component since you aren't using state or lifecycle methods

`export default const BillingView = () => (
      <StripeProvider apiKey="pk_test_TYooMQauvdEDq54NiTphI7jx">
        <h1>Billing View</h1>
        <div className="example">
      </div>
          <h1>React Stripe Elements</h1>
          <Elements>
            <CheckoutForm />
          </Elements>
        </div>
      </StripeProvider>
)